modules/dr-recover-expired-control-plane-certs.adoc

```diff
@@ -34,3 +34,25 @@ $ oc describe csr <csr_name> <1>
 ----
 $ oc adm certificate approve <csr_name>
 ----
+
+In case your load balancer certs are also expired you will need to talk to one of the masters directly.
```

Comment:
Can you give me more detail about what this new content is addressing? Is it just if your load balancer certs are also expired, then you need to set up this SSH tunnel, and then pass in the `--server=...` parameter to all of the `oc` commands in the previous steps?

But then after this procedure is complete, and they've recovered, they don't need to use that `--server=...` parameter anymore?

Is there anything else that they need to do (that we should mention) about them recovering from the expired load balancer certs?